src/main/java/org/zalando/nakadi/service/EventTypeService.java

```diff
@@ -546,7 +546,8 @@ private void repartitionEventType(
                     newStatistics.getWriteParallelism()));
             return;
         }
-        if (existingStatistics.equals(newStatistics)) {
+        if (existingStatistics.getReadParallelism().equals(newStatistics.getReadParallelism())
```

Comment:
you have the same check on line https://github.com/zalando/nakadi/pull/1102/files#diff-0cc0d4406e77f42ca8a2ee3e3ce445e1R569

also on line https://github.com/zalando/nakadi/pull/1102/files#diff-0cc0d4406e77f42ca8a2ee3e3ce445e1R544
is a mistake, it should not trigger repartition from 1 partition to 1

Reply:
Right, fixed in `0ec119d`